docs/source-2.0/spec/resource-traits.rst

```diff
@@ -2,6 +2,111 @@
 Resource traits
```

Comment:
I think the docs for the `resourceIdentifier` trait need to be updated too since it's now used with output, right?

Reply:
Made a light revision. I just removed the wording that suggested it could only be applied to input rather than saying "input or output."